commands/netobserv

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+source "./scripts/functions.sh"
+
+# interface filter such as 'br-ex' or pcap filter such as 'tcp,80'
+filter=""
+
+# CLI image to use
+img="quay.io/netobserv/network-observability-cli:main"
+
+# version to display
+version="0.0.1"
+
+# command to run
+command=""
+
+if [ -z "${1:-}" ] || [ "$1" = "help" ]; then
+    # display Help
+    echo
+    echo "Netobserv allows you to capture flow and packets from your cluster."
+    echo "Find more information at: https://github.com/netobserv/network-observability-cli/"
+    echo
+    echo "Syntax: netobserv [flows|packets|cleanup] [filters]"
+    echo
+    echo "options:"
+    echo "  flows      Capture flows information. You can specify an optionnal interface name as filter such as 'netobserv flows br-ex'."
+    echo "  packets    Capture packets from a specific protocol/port pair such as 'netobserv packets tcp,80'"
+    echo "  cleanup    Remove netobserv components."
+    echo "  version    Print software version."
+    echo
+    exit 0
+elif [ "$1" = "version" ]; then
```

Comment:
pls change this to
```sh
case "$1" in
"version")
    echo "Netobserv CLI version $version"
    exit 0 ;;
"flows")
    ;;
"packets")
    ;;
"cleanup")
    ;;
esac
```

Reply:
done in https://github.com/netobserv/network-observability-cli/pull/27/commits/4e9ea0b7f492e7018a3e83666206ba3fea635708